airflow/providers/microsoft/mssql/hooks/mssql.py

```diff
@@ -51,6 +51,13 @@ def get_conn(
         )
         return conn
 
+    def get_uri(self) -> str:
+        from urllib.parse import urlsplit, urlunsplit
+
+        r = list(urlsplit(super().get_uri()))
+        r[0] = "mssql+pymssql"  # fix to use pymssql driver
+        return urlunsplit(r)
```

Comment:
Why?
This is what we have in `OdbcHook`

https://github.com/apache/airflow/blob/46bbfdade0638cb8a5d187e47034b84e68ddf762/airflow/providers/odbc/hooks/odbc.py#L34

https://airflow.apache.org/docs/apache-airflow-providers-odbc/stable/connections/odbc.html#configuring-the-connection